src/pairings.rs

```diff
@@ -190,14 +190,14 @@ impl_add_binop_specify_output!(MillerLoopResult, MillerLoopResult, MillerLoopRes
 impl AddAssign<MillerLoopResult> for MillerLoopResult {
     #[inline]
     fn add_assign(&mut self, rhs: MillerLoopResult) {
-        *self = &*self + &rhs;
+        *self = *self + rhs;
```

Comment:
Is this going to correctly pick up the `impl Add<&MillerLoopResult> for &MillerLoopResult`? I believe that is what the `&`s were trying to ensure was invoked.

Reply:
Ah, this is fine, because I thought the alternative was recursion, but this is `AddAssign`, so calling any `Add` impl is fine.